packages/components/src/z-stack/component.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { css, cx } from 'emotion';
+// eslint-disable-next-line no-restricted-imports
+import type { Ref } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { isValidElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getValidChildren } from '../ui/utils/get-valid-children';
+import { contextConnect, useContextSystem } from '../ui/context';
+// eslint-disable-next-line no-duplicate-imports
+import type { ViewOwnProps } from '../ui/context';
```

Comment:
Is it a convention in this package to separate type imports from other imports, even if they come from the same file?

```suggestion
import { contextConnect, useContextSystem, ViewOwnProps } from '../ui/context';
```

Reply:
Yup! We actually enforce this through the compiler: https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues

Reply:
Gotcha! It just feels a bit _dirty_ to disable one `eslint` rule in order not to fail another `ts` rule, but definitely not something in the scope of this PR.

Reply:
Sure. 99% of the time it's contained to these components files and won't matter elsewhere. @gziolo how would you feel about disabling this particular rule (`no-duplicate-imports`) for TypeScript files in `components`?